scripts/configure-gitops.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+ARGOCD_INSTANCE_PROVIDED=${ARGOCD_INSTANCE_PROVIDED:-false}
+NAMESPACE=${NAMESPACE:-"ai-rhdh"}
+RHDH_DEPLOYMENT="backstage-ai-rh-developer-hub" # deployment created by rhdh operator by default
+DEFAULT_PLUGIN_CONFIGMAP="backstage-dynamic-plugins-ai-rh-developer-hub" # configmap created by rhdh operator for plugins by default
+DEFAULT_SECRET_NAME="rhdh-argocd-secret"
+
+# ArgoCD Instance Created By Installer
+if [[ $ARGOCD_INSTANCE_PROVIDED == "false" ]]; then
+    # Add ConfigMap To Configure ArgoCD
+    kubectl apply -n $NAMESPACE -f ./resources/argocd-config.yaml
```

Comment:
Considering when we are not on the base directory path, we could use `BASE_DIR="$(realpath $(dirname ${BASH_SOURCE[0]}))/.."` which would make something like this `kubectl apply -n $NAMESPACE -f $BASE_DIR/resources/argocd-config.yaml` removing the need to be in the base directory path to run this script.